src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -1071,29 +1027,31 @@ namespace
 
         try
         {
-            if (extension == ".exr")
-            {
-                write_exr_image(
-                    bf_file_path,
-                    image,
-                    image_attributes);
+            Image transformed_image(image);
+
+            // todo: we may want to be more specific here.
+            if (image.properties().m_channel_count == 4)
+                transform_to_srgb(transformed_image);
```

Comment:
That's wrong, we don't want to convert images to sRGB when writing OpenEXR (and other high dynamic range) files.